gamestonk_terminal/cryptocurrency/overview/withdrawalfees_model.py

```diff
@@ -0,0 +1,150 @@
+            headers={"User-Agent": get_user_agent()},
+        ).text,
+        "lxml",
+    )
+    table = withdrawal_fees_homepage.find_all("table")
+    tickers_html = withdrawal_fees_homepage.find_all("div", {"class": "name"})
+    if table is None or tickers_html is None:
+        return pd.DataFrame()
+    df = pd.read_html(str(table))[0]
+
+    df["Coin"] = [ticker.text for ticker in tickers_html]
+    df["Highest"] = df["Highest"].apply(
+        lambda x: f'{x[:x.index(".")+3]} ({x[x.index(".")+3:]})'
+        if "." in x and isinstance(x, str)
+        else x
+    )
+
+    num_pages = int(math.ceil(top / COINS_PER_PAGE))
+    if num_pages > 1:
+        for idx in range(2, num_pages + 1):
+            withdrawal_fees_homepage = BeautifulSoup(
+                requests.get(
+                    f"https://withdrawalfees.com/coins/page/{idx}",
+                    headers={"User-Agent": get_user_agent()},
+                ).text,
+                "lxml",
+            )
+            table = withdrawal_fees_homepage.find_all("table")
+            tickers_html = withdrawal_fees_homepage.find_all("div", {"class": "name"})
+            if table is not None and tickers_html is not None:
+                new_df = pd.read_html(str(table))[0]
+                new_df["Highest"] = new_df["Highest"].apply(
+                    lambda x: f'{x[:x.index(".")+3]} ({x[x.index(".")+3:]})'
+                    if "." in x
+                    else x
+                )
+                new_df["Coin"] = [ticker.text for ticker in tickers_html]
+                df = df.append(new_df)
+    df = df.fillna("")
+    return df
+
+
+def get_overall_exchange_withdrawal_fees() -> pd.DataFrame:
+    """Scrapes exchange withdrawal fees
+    [Source: https://withdrawalfees.com/]
+
+    Parameters
+    ----------
+
+    Returns
+    -------
+    pandas.DataFrame:
+        Exchange, Coins, Lowest, Average, Median, Highest
+    """
+    exchange_withdrawal_fees = BeautifulSoup(
+        requests.get(
+            "https://withdrawalfees.com/exchanges",
+            headers={"User-Agent": get_user_agent()},
+        ).text,
+        "lxml",
+    )
+    table = exchange_withdrawal_fees.find_all("table")
+    if table is None:
+        return pd.DataFrame()
+    df = pd.read_html(str(table))[0]
+    df = df.fillna("")
+    return df
+
+
+def get_crypto_withdrawal_fees(
+    symbol: str,
+) -> List[Any]:
+    """Scrapes coin withdrawal fees per exchange
+    [Source: https://withdrawalfees.com/]
+
+    Parameters
+    ----------
+    symbol: str
+        Coin to check withdrawal fees. By default bitcoin
+    Returns
+    -------
+    List:
+        - str:              Overall statistics (exchanges, lowest, average and median)
+        - pandas.DataFrame: Exchange, Withdrawal Fee, Minimum Withdrawal Amount
+    """
+    crypto_withdrawal_fees = BeautifulSoup(
+        requests.get(
+            f"https://withdrawalfees.com/coins/{symbol}",
```

Comment:
Check this out. The source uses a symbol, but when we issue a command to the terminal we use a full name and not the symbol
![image](https://user-images.githubusercontent.com/11668535/145102424-d793476b-c173-4d27-ad50-dca05fd219df.png)

To me personally the symbol is preferable, although sometimes I don't know the symbol for a coin and i need to google it.

Is there an option to make the command accept either symbol or a full name? This is more of a question than a proposal

Reply:
Yea, the autocompletion will help here.

I agree that accepting both the symbol or full name would be beneficial not only for this but across some more crypto commands! It will need some work but it is possible

Reply:
why doesn't github let me choose other emojis as a reaction? 🤷‍♂️